### PR TITLE
Remove update-dependencies from API functionality

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

#196 
Completely removes the ability to update dependencies from an API endpoint.
Alternatively, we could still support the ability to use an API endpoint instead of file, and just remove the dep-server default setting. I removed it here, because that extra functionality didn't feel necessary, but I am open to adding it back if needed.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: commands/update_dependencies.go [hunk 1]
```diff
@@ -53,8 +55,6 @@ func updateDependenciesRun(flags updateDependenciesFlags) error {
 		originalVersions[d.Version] = d.Version
 	}
 
-	api := flags.api
-
 	// if a metadata file is provided, use that as the version and metadata
 	// source
 	if flags.metadataFile != "" {
```
Comment (robdimsdale) [inline commands/update_dependencies.go:60]: @sophiewigmore if `metadata-file` is required, can we drop the `if` condition too?

**Review by robdimsdale — DISMISSED** [review]
Looks good. One minor question / potential clean up.

**Review by robdimsdale — APPROVED** [review]

**Comment by robdimsdale** [comment]
FYI the lint/test failures are because of a line that should be removed along with the tests: `Error: integration/init_test.go:35:31: undefined: testUpdateDependencies`

**Comment by robdimsdale** [comment]
LGTM @sophiewigmore - feel free to merge whenever you're ready

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**